lemminx-liberty/src/main/java/io/openliberty/lemminx/liberty/services/FeatureService.java

```diff
@@ -96,23 +109,24 @@ private FeatureService() {
 
     ArrayList<Feature> publicFeatures = new ArrayList<>();
     Arrays.asList(featureList).stream()
-            .filter(f -> f.getWlpInformation().getVisibility().equals(LibertyConstants.PUBLIC_VISIBILITY))
-            .forEach(publicFeatures::add);
+        .filter(f -> f.getWlpInformation().getVisibility().equals(LibertyConstants.PUBLIC_VISIBILITY))
+        .forEach(publicFeatures::add);
     defaultFeatureList = publicFeatures;
     return publicFeatures;
   }
 
-  public List<Feature> getFeatures(String libertyVersion, int requestDelay) {
+  public List<Feature> getFeatures(String libertyVersion, int requestDelay, String documentURI) {
```

Comment:
Maybe add a comment above on how this function works.

So tries to fetch the list of all possible features from maven for the specific liberty version. If that fails, tries to determine the installed features. Else fallback to a hardcoded default list.